src/main/resources/assets/andromeda/lang/zh_cn.json

```diff
@@ -145,6 +153,7 @@
   "text.autoconfig.andromeda.option.campfireTweaks.campfireEffectsPassive": "友善生物治愈",
   "text.autoconfig.andromeda.option.campfireTweaks.campfireEffectsPassive.@Tooltip": "若关闭，治愈效果将仅作用于玩家",
   "text.autoconfig.andromeda.option.campfireTweaks.campfireEffectsList": "你想要生效的药水效果列表",
+  "text.autoconfig.andromeda.option.campfireTweaks.campfireEffectsList": "将你希望能给予的效果列在这里",
```

Comment:
The key should be `text.autoconfig.andromeda.option.campfireTweaks.effectList`

Reply:
oh, sorry